component/web/security/src/main/java/org/exoplatform/web/login/LoginHandler.java

```diff
@@ -392,8 +392,11 @@ private void extendUIParameters(ControllerContext controllerContext, LoginStatus
       String forgotPasswordPath = passwordRecoveryService.getPasswordRecoverURL(null, null);
       params.put("forgotPasswordPath", request.getContextPath() + forgotPasswordPath);
 
-      if (status != LoginStatus.AUTHENTICATED && status != LoginStatus.UNAUTHENTICATED) {
-        params.put("errorCode", status.getErrorCode());
+      params.put("authenticationTitle", PropertyManager.getProperty("portal.authentication.title"));
+      params.put("authenticationSubtitle", PropertyManager.getProperty("portal.authentication.subtitle"));
```

Comment:
Should theses lines be added in other handler ? 
Currently the login page does correctly use properties set in exo.properties, but page forgot password and onboarding user display Title/SubTitle

Reply:
Done, thanks.